### ISSUE TITLE
Does not build on Node v10.2.0

### ISSUE THREAD

**Issue by goyney** [eqyiel/node-woff2#7]
This module no longer builds on Node v10.2.0. It previously built on Node v10.1.0 without issue.

```
error /Users/mirigoyen/Git/myproject/node_modules/woff2: Command failed.
Exit code: 1
Command: node-gyp rebuild
Arguments:
Directory: /Users/mirigoyen/Git/myproject/node_modules/woff2
Output:
gyp info it worked if it ends with ok
gyp info using node-gyp@3.6.2
gyp info using node@10.2.0 | darwin | x64
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp info spawn /usr/local/bin/python2
gyp info spawn args [ '/Users/mirigoyen/Git/myproject/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/mirigoyen/Git/myproject/node_modules/woff2/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/mirigoyen/Git/myproject/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/mirigoyen/.node-gyp/10.2.0/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/mirigoyen/.node-gyp/10.2.0',
gyp info spawn args   '-Dnode_gyp_dir=/Users/mirigoyen/Git/myproject/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/mirigoyen/.node-gyp/10.2.0/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/mirigoyen/Git/myproject/node_modules/woff2',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.' ]
gyp WARN download NVM_NODEJS_ORG_MIRROR is deprecated and will be removed in node-gyp v4, please use NODEJS_ORG_MIRROR
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  CXX(target) Release/obj.target/woff2_decode/woff2/src/font.o
  CXX(target) Release/obj.target/woff2_decode/woff2/src/glyph.o
  CXX(target) Release/obj.target/woff2_decode/woff2/src/normalize.o
  CXX(target) Release/obj.target/woff2_decode/woff2/src/table_tags.o
  CXX(target) Release/obj.target/woff2_decode/woff2/src/transform.o
  CXX(target) Release/obj.target/woff2_decode/woff2/src/variable_length.o
  CXX(target) Release/obj.target/woff2_decode/woff2/src/woff2_common.o
  CXX(target) Release/obj.target/woff2_decode/woff2/src/woff2_dec.o
  CXX(target) Release/obj.target/woff2_decode/woff2/src/woff2_enc.o
  CXX(target) Release/obj.target/woff2_decode/woff2/src/woff2_info.o
  CXX(target) Release/obj.target/woff2_decode/woff2/src/woff2_out.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/common/dictionary.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/dec/bit_reader.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/dec/decode.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/dec/huffman.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/dec/state.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/backward_references.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/backward_references_hq.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/bit_cost.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/block_splitter.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/brotli_bit_stream.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/cluster.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/compress_fragment.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/compress_fragment_two_pass.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/dictionary_hash.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/encode.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/entropy_encode.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/histogram.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/literal_cost.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/memory.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/metablock.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/static_dict.o
  CC(target) Release/obj.target/woff2_decode/woff2/brotli/c/enc/utf8_util.o
  CXX(target) Release/obj.target/woff2_decode/src/woff2_decode.o
In file included from ../src/woff2_decode.cc:2:
In file included from ../../nan/nan.h:51:
/Users/mirigoyen/.node-gyp/10.2.0/include/node/node.h:53:10: fatal error: 'core.h' file not found
#include "core.h"  // NOLINT(build/include_order)
         ^~~~~~~~
1 error generated.
make: *** [Release/obj.target/woff2_decode/src/woff2_decode.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/mirigoyen/Git/myproject/node_modules/node-gyp/lib/build.js:258:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:237:12)
gyp ERR! System Darwin 17.5.0
gyp ERR! command "/Users/mirigoyen/.nvm/versions/node/v10.2.0/bin/node" "/Users/mirigoyen/Git/myproject/node_modules/.bin/node-gyp" "rebuild"
gyp ERR! cwd /Users/mirigoyen/Git/myproject/node_modules/woff2
```

**Comment by eqyiel** [comment]
Thanks, I'll check it out as soon as I get a chance.

**Comment by goyney** [comment]
@eqyiel It actually looks like it was an issue with the Node release. They released a patch version (10.2.1) today and this builds on it.

Perhaps something in their changelog can point to what the issue was: https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V10.md#10.2.1

Since this builds now on 10.2.1, I'm going to close this. Thanks!
